images/70-qemu/create-qemu-disk.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+mkdir /output
+
+guestfish <<__EOF__
+disk-create /output/k3os-qemu.qcow2 qcow2 734003200 preallocation:sparse
+add-drive /output/k3os-qemu.qcow2 name:/dev/sda
+
+disk-create "temp.img" raw 150000000 preallocation:sparse
+add-drive "temp.img"
+
+set-network true
+
+launch
+
+part-disk /dev/sdb mbr
+mkfs ext4 /dev/sdb1
+mount /dev/sdb1 /
+tar-in /alpine-minirootfs-3.11.5-x86_64.tar.gz / compress:gzip
```

Comment:
Why do we need this?

Reply:
In my initial approach as we are not booting from the ISO, getting the alpine mini rootfs was simple than unsquashing and mounting the k3os rootfs.

I am working on a revision that avoids libguestfs altogether and directly boots a patched ISO in qemu.